src/Explorer/Controls/Settings/SettingsSubComponents/IndexingPolicyComponent.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { shallow } from "enzyme";
+import React from "react";
+import { IndexingPolicyComponent, IndexingPolicyComponentProps } from "./IndexingPolicyComponent";
+import { StatefulValue } from "../../StatefulValue";
+import * as DataModels from "../../../../Contracts/DataModels";
+
+describe("IndexingPolicyComponent", () => {
+  const props: IndexingPolicyComponentProps = {
+    shouldDiscardIndexingPolicy: false,
+    resetShouldDiscardIndexingPolicy: () => {
+      return;
+    },
+    indexingPolicyContent: new StatefulValue<DataModels.IndexingPolicy>({
+      automatic: false,
+      indexingMode: "",
+      includedPaths: [],
+      excludedPaths: []
+    }),
+    setIndexingPolicyElementFocussed: () => {
+      return;
```

Comment:
remove

Reply:
having it defined as "setIndexingPolicyElementFocussedL () => {}" gave linting errors , and "() => void" definition isn't possible.

Reply:
There's a typo in your comment, I'm assuming you meant `:` instead of `L`.
What linting error are you getting? if it's [typescript no-empty-function](https://eslint.org/docs/rules/no-empty-function), do this: 
```
setIndexingPolicyElementFocussed: () => {
  // do nothing
}
```